src/test/java/com/fasterxml/jackson/databind/deser/dos/DeepJsonNodeDeser3397Test.java

```diff
@@ -13,7 +13,8 @@ public class DeepJsonNodeDeser3397Test extends BaseMapTest
     // ... currently gets a bit slow at 1M but passes.
     // But test with 100k as practical limit, to guard against regression
 //    private final static int TOO_DEEP_NESTING = 1_000_000;
-    private final static int TOO_DEEP_NESTING = 10_000;
+    private final static int TOO_DEEP_NESTING = StreamReadConstraints.DEFAULT_MAX_DEPTH * 10 < 0
```

Comment:
we are not going to make StreamReadConstraints.DEFAULT_MAX_DEPTH <=0 - this complicated statement is unnecessary - use `TOO_DEEP_NESTING = StreamReadConstraints.DEFAULT_MAX_DEPTH * 10` - this goes for all  the MAX values